nuitka/tools/release/Documentation.py

```diff
@@ -28,68 +28,65 @@
 from nuitka.utils.FileOperations import getFileList
 
 
+def which(prog):
+    """
+    cross version program detection.
+
+    Whenever Nuitka decides to drop Python versions older than 3.3
+    the code in the else block can be removed.
+    """
+    def real(f):
+        def wrap(*args):
+            cmd = ""
+            if sys.version_info > (3, 3):
+                import shutil
+                cmd = shutil.which(prog)
+            else:
+                _, cmdname = os.path.split(prog)
```

Comment:
This looks like `os.path.basename`, but I don't know why accept paths as input, seems unused.

Reply:
That's correct. I just coppied old code of mine which was never reviewed. Thanks for that.